docs/content/guides/getting-started/grid-size.md

```diff
@@ -65,21 +81,31 @@ Handsontable observes window resizing. If the window's dimensions have changed,
 
 You can easily overwrite this behaviour by returning `false` in the [`beforeRefreshDimensions`](@/api/hooks.md#beforerefreshdimensions) hook.
 
+::: only-for javascript
 ```js
 {
   beforeRefreshDimensions() { return false; }
 }
 ```
+:::
+
+::: only-for react
+```jsx
+  <HotTable beforeRefreshDimensions={() => false} /> 
+```
+:::
 
 ## Manual resizing
 
 The Handsontable instance exposes the [`refreshDimensions()`](@/api/core.md#refreshdimensions) method, which helps you to resize grid elements properly.
 
+::: only-for javascript
```

Comment:
It don't see a good reason to hide this snippet from React readers.

Reply:
Will change